.circleci/config.yml

```
@@ -1,17 +1,53 @@
-version: 2
-jobs:
+version: 2.1
+
+orbs:
+  architect: giantswarm/architect@0.8.14
+
+workflows:
   build:
-    machine: true
-    steps:
-    - checkout
+    jobs:
+      - architect/go-build:
+          name: go-build
+          binary: ingress-exporter
+          filters:
+            tags:
+              only: /^v.*/
+
+      - architect/push-to-docker:
+          context: architect
+          name: push-ingress-exporter-to-quay
+          image: "quay.io/giantswarm/ingress-exporter"
+          username_envar: "QUAY_USERNAME"
+          password_envar: "QUAY_PASSWORD"
+          requires:
+            - go-build
+          filters:
+            # Trigger the job also on git tag.
+            tags:
+              only: /^v.*/
+
+      - architect/push-to-app-catalog:
+          context: architect
+          name: push-ingress-exporter-to-control-plane-app-catalog
+          app_catalog: "control-plane-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "ingress-exporter"
+          requires:
+            - push-ingress-exporter-to-quay
+          filters:
+            tags:
+              only: /^v.*/
 
-    - run: |
-        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/tags/v1.0.0 | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-    - run: ./architect build
-    - deploy:
-        command: |
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            ./architect deploy
-          fi
+      - architect/push-to-app-collection:
+          context: architect
+          name: push-ingress-exporter-to-kvm-app-collection
+          app_name: "ingress-exporter"
```

I forgot we need to add this:

```
```suggestion
          app_namespace: "monitoring"
```

---

Done done